internal/commands/move/move.go

```diff
@@ -0,0 +1,142 @@
+package move
+
+import (
+	"errors"
+
+	"github.com/NilsPonsard/verbosity"
+	"github.com/bwmarrin/discordgo"
+	"github.com/do3-2021/booty-mover/internal/commands/common"
+)
+
+var command = &discordgo.ApplicationCommand{
+	Name:        "move",
+	Description: "Move all users from one voice channel to another",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Name:         "source_channel",
+			Type:         discordgo.ApplicationCommandOptionChannel,
+			ChannelTypes: []discordgo.ChannelType{discordgo.ChannelTypeGuildVoice},
+			Description:  "Source channel (voice channel)",
+			Required:     true,
+		},
+		{
+			Name:         "destination_channel",
+			Type:         discordgo.ApplicationCommandOptionChannel,
+			ChannelTypes: []discordgo.ChannelType{discordgo.ChannelTypeGuildVoice},
+			Description:  "Destination channel (voice channel)",
+			Required:     true,
+		},
```

Comment:
Maybe it would be simpler to write 

"Source voice channel"
and
"Destination voice channel"